src/Defaults.php

```diff
@@ -358,4 +363,5 @@ public function maxItems($value = null)
     private $scrubFields;
     private $customTruncation = null;
     private $maxItems = 10;
+    private $minimumLevel = null;
```

Comment:
We also should probably default this to `\Monolog\Logger::error`

Reply:
Yes it's good =)